walkthroughs/http-headers-and-priority/header.yaml

```diff
@@ -41,6 +41,7 @@ Resources:
         }
       ManagedPolicyArns:
       - arn:aws:iam::aws:policy/CloudWatchFullAccess
+      - arn:aws:iam::aws:policy/AWSAppMeshEnvoyAccess
```

Comment:
For everything under `walkthroughs`, could we also add `AWSAppMeshPreviewEnvoyAccess`?

We use separate managed policies for preview vs. production, and all the walk-throughs use preview.